crt_portal/cts_forms/forms.py

```diff
@@ -875,6 +876,20 @@ class Meta:
             'intake_filters': 'View sections'
         }
 
+    def clean_filters(self, cleaned_data):
```

Comment:
Since we're relying on Django's form validation here, there's an opportunity to use a field specific clean function and not have to redefine `clean` as well.

https://docs.djangoproject.com/en/2.2/ref/forms/validation/#form-and-field-validation
Checkout the snippet there on: "The clean_fieldname() method"